src/dynamodb.rs

```diff
@@ -156,6 +161,26 @@ macro_rules! val {
 	);
 }
 
+// TODO: make a macro from this?
+pub fn get_string_from_attribute(attr: &AttributeValue) -> Option<&str> {
```

Comment:
Shouldn't this be "get_str..." instead of string?  Leftovers from when I had this returning a String instead of &str.